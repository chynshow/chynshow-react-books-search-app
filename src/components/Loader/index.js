import React from 'react';

//   Loader https://tobiasahlin.com
const Loader = () => (
  <div className='sk-chase'>
    <div className='sk-chase-dot'></div>
    <div className='sk-chase-dot'></div>
    <div className='sk-chase-dot'></div>
    <div className='sk-chase-dot'></div>
    <div className='sk-chase-dot'></div>
    <div className='sk-chase-dot'></div>
  </div>
);

export default Loader;
