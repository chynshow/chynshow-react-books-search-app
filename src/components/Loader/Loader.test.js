import React from 'react';
import Loader from './';
import { shallow } from 'enzyme';

describe('Loader component', () => {
  it('should render Loader component', () => {
    const c = shallow(<Loader />);
    expect(c).toMatchSnapshot();
  });
});
