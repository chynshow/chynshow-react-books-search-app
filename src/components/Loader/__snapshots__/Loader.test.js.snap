// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render Loader component 1`] = `
<div
  className="sk-chase"
>
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
</div>
`;
